app/services/workflows/non_passported_workflow.rb

```diff
@@ -228,6 +228,29 @@ def calculate_partner_childcare_eligibility(assessment, applicant, partner)
       def partner_allowance(submission_date)
         Threshold.value_for(:partner_allowance, at: submission_date)
       end
+
+      def vehicles(vehicles, disputed: true)
+        (disputed ? vehicles.select(&:subject_matter_of_dispute) : vehicles.reject(&:subject_matter_of_dispute)).map do |vehicle|
+          Assessors::VehicleAssessor::VehicleData.new(vehicle:, result: Assessors::VehicleAssessor::Result.new(assessed_value: 0, included_in_assessment: false))
+        end
+      end
+
+      def person_capital_subtotals(person)
+        person_vehicles = person&.vehicles || []
+        PersonCapitalSubtotals.new(disputed_vehicles: vehicles(person_vehicles, disputed: true), non_disputed_vehicles: vehicles(person_vehicles, disputed: false), total_liquid: 0,
+                                   total_mortgage_allowance: 0.0, total_non_liquid: 0.0,
+                                   disputed_property_disregard: 0.0, pensioner_capital_disregard: 0.0,
+                                   properties: [], disputed_non_property_disregard: 0.0, disputed_non_property_capital: 0.0, non_disputed_non_property_capital: 0.0)
+      end
+
+      def dummy_capital_subtotals(applicant:, partner:)
+        CapitalSubtotals.new(
+          applicant_capital_subtotals: person_capital_subtotals(applicant),
+          partner_capital_subtotals: person_capital_subtotals(partner),
+          capital_contribution: 0,
+          combined_assessed_capital: 0,
+        )
+      end
```

Comment:
I think it would be nice if these were methods on CapitalSubtotals and PersonCapitalSubtotals. I think they should be replacements for the blank() method - slightly worried that our code coverage didn't complain.